unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioClip.cs

```diff
@@ -42,6 +47,12 @@ public DCLAudioClip(IParcelScene scene) : base(scene)
             loadingState = LoadState.IDLE;
         }
 
+        public double Volume => ((Model)model).volume;
+
+        public bool IsLoop => ((Model)model).loop;
+        
+        public bool ShouldTryLoad => ((Model)model).shouldTryToLoad;
```

Comment:
This naming doesn't match with the style on the rest of our codebase, we should switch to lower camelCase here.